src/modules/did/did.service.ts

```diff
@@ -142,11 +142,11 @@ export class DIDService {
   }
 
   /**
-   * Refresh the DID Document cache for a given DID.
+   * Fetch updates from IPFS for a given DID document.
```

Comment:
The DID document updates are obtained from the blockchain, not IPFS
```suggestion
   * Add any incremental changes to the DID document that occurred since the last sync.
```